src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -505,11 +505,11 @@ private static Transform getTransform(Service s,
      *
      * @throws NoSuchAlgorithmException if {@code transformation}
      *         is {@code null}, empty, in an invalid format,
```

Comment:
Not related to this change, but do we need an "or" before "in an valid format"?

Reply:
A semi-colon (instead of a comma) after "format" might also help separate the 2 main error conditions:

@throws NoSuchAlgorithmException if {@code transformation}
        is {@code null}, empty or in an invalid format;
        or if a {@code CipherSpi} implementation is not found or
        is found but does not support the mode

Reply:
Sure, will use this. Thanks for the suggestions @wangweij @seanjmullan